week_06/case_study_06.Rmd

```diff
@@ -1,7 +1,41 @@
 ---
 title: "Case Study 06"
-author: Your Name
-date: August 1, 2020
+author: Willow Flood
+date: October 11, 2022
 output: github_document
 ---
- 
\ No newline at end of file
+
+library(raster)
```

Comment:
if you are saving this as a .Rmd, all code should be in code chunks. See the help for rmarkdown.